pkg/scheduler/scheduler_test.go

```diff
@@ -132,35 +132,83 @@ func TestSchedulerCreation(t *testing.T) {
 		wantProfiles []string
 	}{
 		{
-			name:         "default scheduler",
-			wantProfiles: []string{"default-scheduler"},
-		},
-		{
-			name:         "valid out-of-tree registry",
-			opts:         []Option{WithFrameworkOutOfTreeRegistry(validRegistry)},
+			name: "valid out-of-tree registry",
```

Comment:
Can we add a subtest to cover that an out-of-tree with its custom pluginArgs?

Reply:
We have in scheme_test.go.